api/protobuf/core/api_container_service.proto

```diff
@@ -176,6 +176,9 @@ message RunStarlarkScriptArgs {
 
   // Defaults to 4
   optional int32 parallelism = 4;
+
+  // The name of the main function, the default value is "run"
```

Comment:
If there a default, should we make this argument optional? Or we consider empty string means no value? Maybe that's fine

Reply:
the empty string here means use the default value, which is 'run'

I think both versions are the same because when you use an `optional string` like for example `optional string subnetwork ` then when you call the `GetSubnetwork()` it returns an empty string if the value is nil. 

<img width="339" alt="image" src="https://github.com/kurtosis-tech/kurtosis/assets/29951623/1c6a0313-ce60-4d03-a7ad-aa690461fbd6">

<img width="391" alt="image" src="https://github.com/kurtosis-tech/kurtosis/assets/29951623/f5e59768-923e-4f80-aa85-5a39304721ae">

but I could be missing some pattern here, so i'm all ears